ui/mirage/models/variable.ts

```diff
@@ -0,0 +1,24 @@
+import { Model, belongsTo, hasMany } from 'ember-cli-mirage';
+import { Variable, Ref } from 'waypoint-pb';
```

Comment:
Can we remove these unused imports?

```suggestion
import { Model, belongsTo } from 'ember-cli-mirage';
import { Variable } from 'waypoint-pb';
```